.github/workflows/cypress.yml

```diff
@@ -9,4 +9,6 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Component tests
-        run: npm run test:ct
\ No newline at end of file
+        run: npm run test:ct
+      - name: Create coverage
+        run: npm run coverage
```

Comment:
I think you'd need to run both jest and cypress tests within the same job. This way, the coverage command will collect coverage results from both tests

Reply:
Yes, looks like that. The same way was done for ocp-advisor.